src/SourceBuild/content/build.proj

```diff
@@ -9,7 +9,11 @@
     </ProjectReference>
   </ItemDefinitionGroup>
 
-  <ItemGroup>
+  <PropertyGroup>
+    <DotNetBuildPass Condition="'$(DotNetBuildPass)' == ''">1</DotNetBuildPass>
+  </PropertyGroup>
```

Comment:
Why do you need to set this to a default value as part of your change?

Reply:
I don't I just thought we'd probably do something like this at some point. I can remove it

Reply:
Removed